inc/vcf/string_constants.hpp

```diff
@@ -102,6 +102,9 @@ namespace ebi
     const std::string INS = "INS";
     const std::string INV = "INV";
 
+    // ALT for gVCF
+    const std::string GVCF = "<*>";
```

Comment:
@cyenyxe, what about NON_REFERENCE_ALLELE? NON_REFERENCE?

Reply:
not done in nexxt commit

Reply:
I would suggest GVCF_NON_VARIANT_ALLELE because in `record.cpp` we display the following error when dot is listed along with others:

```
The no-alternate alleles symbol (dot) can not be combined with others"};
```

Just to make clear that one is the GVCF definition and the other a more VCF-generic one.